auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -406,7 +427,7 @@ public IdpConfig build() {
     public final class SignInIntentBuilder {
         private int mLogo = NO_LOGO;
         private int mTheme = getDefaultTheme();
-        private LinkedHashSet<IdpConfig> mProviders = new LinkedHashSet<>();
+        private List<IdpConfig> mProviders = new ArrayList<>();
```

Comment:
This is much better, we were converting this to a list anyway which seems kinda pointless.